diff --git b/helm/external-dns-app/templates/_helpers.tpl a/helm/external-dns-app/templates/_helpers.tpl
index 19ce2cb..185594d 100644
--- b/helm/external-dns-app/templates/_helpers.tpl
+++ a/helm/external-dns-app/templates/_helpers.tpl
@@ -37,7 +37,15 @@ If release name contains chart name it will be used as a full name.
 Create chart name and version as used by the chart label.
 */}}
 {{- define "external-dns.chart" -}}
-{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" | trimSuffix "." }}
+{{- end }}
+
+{{/*
+Giantswarm labels
+*/}}
+{{- define "giantswarm.labels" -}}
+giantswarm.io/service-type: "{{ .Values.serviceType }}"
+application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
 {{- end }}

 {{/*
@@ -53,6 +61,7 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- with .Values.commonLabels }}
 {{ toYaml . }}
 {{- end }}
+{{ include "giantswarm.labels" . }}
 {{- end }}

 {{/*
