diff --git a/helm/external-dns-app/templates/_helpers.tpl b/helm/external-dns-app/templates/_helpers.tpl
index a612f81..5da027a 100644
--- a/helm/external-dns-app/templates/_helpers.tpl
+++ b/helm/external-dns-app/templates/_helpers.tpl
@@ -43,6 +43,8 @@ Common labels
 */}}
 {{- define "external-dns.labels" -}}
 helm.sh/chart: {{ include "external-dns.chart" . }}
+app.kubernetes.io/name: {{ include "external-dns.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
 {{ include "external-dns.selectorLabels" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
@@ -58,8 +60,7 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 Selector labels
 */}}
 {{- define "external-dns.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "external-dns.name" . }}
-app.kubernetes.io/instance: {{ .Release.Name }}
+app: {{ .Release.Name | quote }}
 {{- end }}
 
 {{/*
