diff --git a/helm/external-dns-app/README.md.gotmpl b/helm/external-dns-app/README.md.gotmpl
new file mode 100644
index 0000000..3d44fe4
--- /dev/null
+++ b/helm/external-dns-app/README.md.gotmpl
@@ -0,0 +1,12 @@
+{{ template "chart.header" . }}
+{{ template "chart.description" . }}
+
+{{ template "chart.versionBadge" . }}
+{{ template "chart.typeBadge" . }}
+{{ template "chart.appVersionBadge" . }}
+
+{{ template "chart.homepageLine" . }}
+
+{{ template "chart.requirementsSection" . }}
+
+{{ template "chart.valuesSection" . }}
