diff --git a/vendor/external-dns/templates/_helpers.tpl b/helm/external-dns-app/templates/_helpers.tpl
index aad0982..5da027a 100644
--- a/vendor/external-dns/templates/_helpers.tpl
+++ b/helm/external-dns-app/templates/_helpers.tpl
@@ -30,11 +30,21 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Giantswarm labels
+*/}}
+{{- define "giantswarm.labels" -}}
+giantswarm.io/service-type: "{{ .Values.serviceType }}"
+application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
+{{- end }}
+
 {{/*
 Common labels
 */}}
 {{- define "external-dns.labels" -}}
 helm.sh/chart: {{ include "external-dns.chart" . }}
+app.kubernetes.io/name: {{ include "external-dns.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
 {{ include "external-dns.selectorLabels" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
@@ -43,14 +53,14 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- with .Values.commonLabels }}
 {{ toYaml . }}
 {{- end }}
+{{ include "giantswarm.labels" . }}
 {{- end }}
 
 {{/*
 Selector labels
 */}}
 {{- define "external-dns.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "external-dns.name" . }}
-app.kubernetes.io/instance: {{ .Release.Name }}
+app: {{ .Release.Name | quote }}
 {{- end }}
 
 {{/*
